src/sample.jl

```diff
@@ -423,20 +427,25 @@ function mcmcsample(
 
             Distributed.@async begin
                 try
-                    chains = Distributed.pmap(pool, seeds) do seed
+                    function sample_chain(seed, init_params=nothing)
```

Comment:
Is the default value necessary, since it is below always called with the argument provided?
```suggestion
                    function sample_chain(seed, init_params)
```

Reply:
It's not called with the argument if `init_params=nothing` - at least it shouldn't.

Reply:
In line 445 `init_params` is not provided since we can't map over it.

Reply:
Oops, yeah, I see it now.